server/services/config/combined_test.go

```diff
@@ -122,23 +122,17 @@ func TestFetchFromConfigService(t *testing.T) {
 		// check signature
 		pubKeyID := "woodpecker-ci-plugins"
 
-		keystore := httpsig.NewMemoryKeyStore()
-		keystore.SetKey(pubKeyID, pubEd25519Key)
+		verifier, err := httpsign.NewEd25519Verifier(pubKeyID, pubEd25519Key,
+			httpsign.NewVerifyConfig(),
+			httpsign.Headers("@request-target", "content-digest")) // The Content-Digest header will be auto-generated
```

Comment:
Not sure if it is internally included, but I think date should be part as well to prevent re-play attacks if I remember correctly.